### ISSUE TITLE
EXO Shell Isssue

### ISSUE THREAD

**Issue by KiranBhaskar1982** [andysvints/ExchangeOnlineShell#9]
I am trying to run EXO Shell from my notebook and currently we have allowed only Signed Scripts to run from local notebook. owever i see that the PS1 (CreateExoPSSession) is signed but still i seeing an error which is attached for your refrence
I Have exported the certificate from  the signed PS1 and have imported the same to Trusted Publisher. Yet the issye persist.

Kindy assist.
![Capture](https://user-images.githubusercontent.com/103553324/163099840-fe4a1d59-47f5-4dfb-b408-1bdc22d0dd82.JPG)

**Comment by andysvints** [comment]
Connect-EXOPSSession is part of ExchangeOnlineManagement module: https://www.powershellgallery.com/packages/ExchangeOnlineManagement/2.0.5

Also there is a new version of connect cmdlet Connect-ExchangeOnline
https://docs.microsoft.com/en-us/powershell/exchange/exchange-online-powershell-v2?view=exchange-ps

Hope that helps.

**Comment by KiranBhaskar1982** [comment]
I am still getting same error when i try to connect to Exchange online shell. I was able to download the EXO Shell from repository and installed in my notebook even that was present earlier. the error appears only when i try to connect to exchange online shell. the reason behind this is that we have disabled unsigned scripts to be running from PS.
![Capture1](https://user-images.githubusercontent.com/103553324/163120156-8e1716d5-6c19-4e19-8e6a-428b86966e39.JPG)
